### ISSUE TITLE
A11y: Edit Panel: The status of some elements is not perceivable in high contrast mode

### ISSUE THREAD

**Issue by ChristofSimon** [grafana/grafana#108066]
**Steps to reproduce**:
Activate high contrast mode of Windows (ALT + SHIFT + PRINT)
Dashboards - Edit Panel

**Actual Result**:
In the following picture there are controls where the actual status is not perceivable (framed light blue).

<img width="1086" height="397" alt="Image" src="https://github.com/user-attachments/assets/ccdd476a-fffd-4083-97d9-39ced2ac69af" />

**Expected Result**
The actual status of all controls should be perceivable also in high contrast mode.
People with a visual impairment often use the option to create their own color scheme in Windows, or use one of the predefined schemes. It is therefore important that the application can also be operated when using the high-contrast mode in Windows. 

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
Success Criterion 1.4.1 Use of Color
https://www.w3.org/TR/WCAG21/#use-of-color

**Environment**:
- Grafana version: 12.0.1
- Data source type & version: Microsoft Windows 11 Enterprise / Chrome 138.0.7204.101
- User OS & Browser:
- Others:
